pymc/distributions/multivariate.py

```diff
@@ -352,42 +352,49 @@ class MvStudentT(Continuous):
     nu : tensor_like of float
         Degrees of freedom, should be a positive scalar.
     Sigma : tensor_like of float, optional
-        Covariance matrix. Use `cov` in new code.
+        Scale matrix. Use `scale` in new code.
     mu : tensor_like of float, optional
         Vector of means.
-    cov : tensor_like of float, optional
-        The covariance matrix.
+    scale : tensor_like of float, optional
+        The scale matrix.
     tau : tensor_like of float, optional
         The precision matrix.
     chol : tensor_like of float, optional
-        The cholesky factor of the covariance matrix.
+        The cholesky factor of the scale matrix.
     lower : bool, default=True
         Whether the cholesky fatcor is given as a lower triangular matrix.
     """
     rv_op = mv_studentt
 
     @classmethod
-    def dist(cls, nu, Sigma=None, mu=None, cov=None, tau=None, chol=None, lower=True, **kwargs):
+    def dist(cls, nu, Sigma=None, mu=None, scale=None, tau=None, chol=None, lower=True, **kwargs):
```

Comment:
I know this isnt part of code change but just curious. Why is Sigma capitalized?

Reply:
Just a convention related to the fact that its a matrix rather than a scalar.